src/ui/display.h

```diff
@@ -165,6 +167,8 @@ void ui_display_output_address_amount_flow(int index);
 
 void ui_display_output_address_amount_no_index_flow(int index);
 
+void ui_warn_high_fee_flow();
```

Comment:
You forget to put `void` for the function parameters :)

Reply:
Fixed in 1e7404b78411f461a923fdcaca9a7226117c0bac